examples/README.md

```diff
@@ -17,25 +17,16 @@ directory.
 ## Cluster Certificates
 
 The easiest way to create certificates is to use the local [cert-operator]
-setup. See [this guide][cert-operator-local-setup] for details. Using the [helm registry plugin]
-the cert-operator charts can be installed easily:
+setup. See [this guide][cert-operator-local-setup] for details.
+
+- Note: `clusterName` and `commonDomain` chart values must match the values used
+  during this guide. Also, you don't need to build cert-operator, just pass `latest`
+  to the `imageTag` value:
 
 ```bash
-helm registry install quay.io/giantswarm/cert-operator-lab-chart -- \
-                   -n cert-operator-lab \
-                   --set imageTag=latest \
-                   --set clusterName=my-cluster \
-                   --set commonDomain=my-common-domain \
-                   --wait
-helm registry install quay.io/giantswarm/cert-resource-lab-chart -- \
-                   -n cert-resource-lab \
-                   --set clusterName=my-cluster \
-                   --set commonDomain=my-common-domain
+helm install -n cert-operator-lab ./cert-operator-lab-chart/ --set imageTag latest --wait
```

Comment:
I'd replace it with:

```
git clone github.com/giantswarm/cert-operator ./cert-operator

helm \
    install -n cert-operator-lab ./cert-operator/examples/cert-operator-lab-chart/ \
    --set clusterName=my-cluster \
    --set commonDomain=my-common-domain \
    --set imageTag latest \
    --wait
```
WDYT?

Reply:
Much better :) I've also added a note about the waiting for the TPR to be ready before trying to install the resources chart, pls take a look.